src/resources/CloudProviderGcpManagedDatabaseInstanceType.yaml

```diff
@@ -0,0 +1,21 @@
+get:
+  summary: 'List GCP available managed database instance types'
+  operationId: listGcpManagedDatabaseInstanceType
+  parameters:
+    - $ref: '../parameters/path/region.yaml'
+    - $ref: '../parameters/path/databaseType.yaml'
+  tags:
+    - Cloud Provider
+  responses:
+    '200':
+      description: 'list GCP available managed database instance types'
```

Comment:
We'll handle manage databases for GCP ?

Reply:
Not for now, but all endpoints are there in Core already, it will return an empty list.

Reply:
Pretty sure those endpoints are never called (e.g for scaleway we don't allow managed db, so users would never be able to choose a database instance type)

Reply:
I removed managed DB instance endpoint and node instance types for the time being